Directory.Build.targets

```
@@ -115,6 +115,19 @@
     <AssemblyVersion>$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion).0.0</AssemblyVersion>
   </PropertyGroup>
 
+  <PropertyGroup Label="APICompat settings">
+    <!-- Don't run APICompat on projects without ref assemblies -->
+    <RunApiCompat Condition="'$(HasReferenceAssembly)' == 'false'">false</RunApiCompat>
```

`$(HasReferenceAssembly)` is never empty (see https://github.com/aspnet/AspNetCore/blob/release/3.1/Directory.Build.targets#L99-L101). All uses of it should be simplified e.g.
```suggestion
    <RunApiCompat Condition=" !$(HasReferenceAssembly) ">false</RunApiCompat>
```